sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -122,6 +124,15 @@ def update_tor_active_side(read_side, state, logical_port_name):
                     return y_cable_toggle_mux_torA(physical_port)
 
             # TODO: Should we confirm that the mux was indeed toggled?
+            # attempt to read the cable direction in 3 attempts and if still the direction is not in sync
+            # return unknown
+            for i in range(Y_CABLE_DIRECTION_SYNC):
```

Comment:
how would that work if the above calls are returning already?

Reply:
fix